src/handlers/compilation/includes.ts

```diff
@@ -131,24 +137,23 @@ const readIncludedFileFromWorkspace = (
   return Promise.resolve(includedFile.getText());
 };
 
-export type FileSystemReader = (filename: Filename) => Promise<FileContent>;
-let fileSystemReader: FileSystemReader | undefined = undefined;
+// export type FileSystemReader = (filename: Filename) => Promise<FileContent>;
```

Comment:
I think these comments can be deleted

Reply:
Yeah, I also had been thinking about putting the types where they are used. I thought each module that has types should have a separete file for the types and export them in the index.ts.